app/views/reviews/show.html.erb

```diff
@@ -20,16 +20,16 @@
         <div class="child">
 
           <div class="grid-pet-container pet">
-            <div class="banner-grid_item"></div>
-
-
             <div class="info">
+
```

Comment:
This is the styling for the review for a specific shelter. We should probably revise this, it looks OK for now.

Reply:
Sounds great